packages/react/src/Dm3.tsx

```diff
@@ -180,7 +180,7 @@ function dm3(props: dm3Props) {
             }
 
             const socket = socketIOClient(
-                state.connection.account.profile.deliveryServiceUrl,
+                state.connection.account.profile.deliveryServices[0],
```

Comment:
`socketIOClient` needs a URL and `deliveryServices[0]` contains the ENS name of the delivery service. 
With the new structure, it is necessary to take the ENS name of the delivery service and get the URL from the delivery service record.